FabricExample/patches/react-native+0.0.0-20220126-2009-86fa2a510.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java
+index 6f86abb..8a91d85 100644
+--- a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java
++++ b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java
+@@ -21,7 +21,7 @@ public class AndroidInfoHelpers {
+ 
+   public static final String EMULATOR_LOCALHOST = "10.0.2.2";
+   public static final String GENYMOTION_LOCALHOST = "10.0.3.2";
+-  public static final String DEVICE_LOCALHOST = "localhost";
++  public static final String DEVICE_LOCALHOST = EMULATOR_LOCALHOST;
+ 
+   public static final String METRO_HOST_PROP_NAME = "metro.host";
+ 
+diff --git a/node_modules/react-native/scripts/.packager.env b/node_modules/react-native/scripts/.packager.env
+new file mode 100644
+index 0000000..361f5fb
+--- /dev/null
++++ b/node_modules/react-native/scripts/.packager.env
+@@ -0,0 +1 @@
++export RCT_METRO_PORT=8081
```

Comment:
This was used for testing purposes, so that it is easier to use this example on M1

Reply:
Indeed, with Hermes enabled the app seems to crash if Metro bundler is not available. In general, `adb reverse tcp:8081 tcp:8081` is the workaround.